app/views/members/new.html.haml

```diff
@@ -1,26 +1,30 @@
 .container.py-4.py-lg-5
   .row.justify-content-md-center
-    .col-md-10.col-lg-8
+    .col
       %h1= @page_title
 
 .py-4.py-lg-5.bg-light
   .container
     .row.justify-content-md-center
```

Comment:
@KimberleyCook we could remove this .row and the .col before and use the .row on line 10.